.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - run:
           name: "Run unit tests"
           command: |
+            docker run conda activate COMPASS
             docker run python -m pytest
```

Comment:
Successive `docker run`s will run in separate containers, so I don't think this first command will have an effect on the second. I'd try my above suggestion first - IME activating the environment isn't always needed. If you still end up needing to activate the env, you'll want to do it in a single run command - I think `docker run mytag bash -c "conda activate COMPASS && docker run python -m pytest"` would work.